plugins/zwave/src/CommandClasses/SettingsToConfiguration.ts

```diff
@@ -14,13 +14,13 @@ export class SettingsToConfiguration extends ZwaveDeviceBase implements Settings
         const node = this.instance.getNodeUnsafe();
         for (var valueId of values) {
             const metadata = node.getValueMetadata(valueId);
-
             let setting: Setting = {};
             setting.key = valueId.property.toString();
             setting.title = metadata.label;
             setting.description = metadata.description;
             if ((metadata as any).states)
                 setting.choices = Object.values((metadata as any).states);
+                setting.combobox = true;
```

Comment:
indent? was this meant to be under the if or always applied?

Reply:
why would you want free text entry? I'm not sure that's valid

Reply:
This was intentional. i.e. "make it a combobox if there are states(choices)"